### PR TITLE
Release 3.2.0-BETA.2

### PR DESCRIPTION
**Pull request by heyams**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # CHANGELOG
 
+# Version 3.2.0-BETA.2
+* There were lots of refactorings.
```
Comment (heyams) [inline CHANGELOG.md:4]: @trask we might need to reword these in the github release notes.

Comment (trask) [reply CHANGELOG.md:4]: we only need to put user facing changes in the change log
```suggestion
* Lots of internal clean up.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+* Enable profiler by default.
```
Comment (trask) [inline CHANGELOG.md:5]: the profiler is still proof-of-concept (and undocumented), so let's not mention it (also, it's not enabled by default, it's only included by default)
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+* Reduce binary size.
```
Comment (trask) [inline CHANGELOG.md:6]: just to clarify that it's not reduced from 3.1.1
```suggestion
* Reduce binary size from 3.2.0-BETA
```

**Review by trask — APPROVED** [review]

**Pull request merged**